tests/Meziantou.Analyzer.Test/Rules/DoNotUseElseWhenIfBlockJumpsAnalyzerTests.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System.Threading.Tasks;
```

Comment:
I think you should add more tests:

````
if (true)
{
    if (true) // nested if
        return;
}
else
{
    // This is not useless
}
````

````
if (true)
{
    return;

    The last statement is a local function (I thinks this test will fail because of the implementation of IsOrEndsWithJumpStatement)
    bool Condition() => return true;
}
else
{
    // This is useless
}
````